### ISSUE TITLE
Delete or archive this repo?

### ISSUE THREAD

**Issue by lucyllewy** [ubuntu/uupgrade-web#1]
This project looks like it never got off the ground. Would it be a good idea to delete or archive the repo?

**Comment by shriekdj** [comment]
Yeah I Think Also But I will Try to use this if it have any use @diddledani
